### PR TITLE
Restrict number persons warning dialog (EXPOSUREAPP-10970)

### PR DESCRIPTION
**Pull request by AlexanderAlferov**
## Whats new
Warning dialogs if number of persons is more than threshold and maximum count of persons certificates.

## Jira
[Ticket](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-10970)

## How to test
Create and scan certificates for new person every time of any type until you would see warning dialog.

### CODE REVIEW

File: Corona-Warn-App/src/main/res/values-de/strings.xml [hunk 1]
```diff
@@ -947,6 +947,22 @@ Sollten Sie den Test in der App gelöscht haben, können Sie ihn aus dem Papierk
     <string name="qr_code_no_suitable_qr_code_title">Kein geeigneter QR-Code</string>
     <!-- XTXT: QR Code error message title  -->
     <string name="qr_code_no_suitable_qr_code_body">Der QR-Code wird nicht von der Corona-Warn-App unterstützt. Bitte wählen Sie einen geeigneten QR-Code.</string>
+    <!-- XTXT: QR Code error message: halfway title (10+ people already scanned)  -->
+    <string name="qr_code_error_max_person_threshold_title">"Warnung"</string>
+    <!-- XTXT: QR Code error message: halfway body (10+ people already scanned)  -->
+    <string name="qr_code_error_max_person_threshold_body">"Die Corona-Warn-App darf nicht zur  Zertifikatsprüfung durch Dritte verwendet werden. Nutzen Sie dafür die CovPassCheck-App.\n\nSie können maximal für %s Personen Zertifikate hinzufügen. Danach können Sie keine neuen Zertifikate mehr für weitere Personen einscannen.\n\nWeitere Informationen zum  Impfzertifikat finden Sie in den FAQ."</string>
```
Comment (chiljamgossow) [inline Corona-Warn-App/src/main/res/values-de/strings.xml:953]: extra space after zur

File: Corona-Warn-App/src/main/res/values-de/strings.xml [hunk 2]
```diff
@@ -947,6 +947,22 @@ Sollten Sie den Test in der App gelöscht haben, können Sie ihn aus dem Papierk
...
+    <!-- XTXT: QR Code error message: halfway title (10+ people already scanned)  -->
+    <string name="qr_code_error_max_person_max_title">"Fehler"</string>
+    <!-- XTXT: QR Code error message: halfway body (10+ people already scanned)  -->
+    <string name="qr_code_error_max_person_max_body">"Die Corona-Warn-App darf nicht zur  Zertifikatsprüfung durch Dritte verwendet werden. Nutzen Sie dafür die CovPassCheck-App.\n\nSie können maximal für 20 Personen Zertifikate hinzufügen. Sie können keine neuen Zertifikate mehr für weitere Personen einscannen.\n\nWeitere Informationen zum  Impfzertifikat finden Sie in den FAQ."</string>
```
Comment (Ein-Tim) [inline Corona-Warn-App/src/main/res/values-de/strings.xml:957]: One space too much.

```suggestion
    <string name="qr_code_error_max_person_max_body">"Die Corona-Warn-App darf nicht zur Zertifikatsprüfung durch Dritte verwendet werden. Nutzen Sie dafür die CovPassCheck-App.\n\nSie können maximal für 20 Personen Zertifikate hinzufügen. Sie können keine neuen Zertifikate mehr für weitere Personen einscannen.\n\nWeitere Informationen zum  Impfzertifikat finden Sie in den FAQ."</string>
```

**Review by Ein-Tim — CHANGES_REQUESTED** [review]

File: Corona-Warn-App/src/main/res/values-de/strings.xml (hunk 2 above)
Comment (mtwalli) [reply Corona-Warn-App/src/main/res/values-de/strings.xml:957]: @KathrinLuecke  FYI

File: Corona-Warn-App/src/main/res/values-de/strings.xml [hunk 3]
```diff
@@ -947,6 +947,22 @@ Sollten Sie den Test in der App gelöscht haben, können Sie ihn aus dem Papierk
...
+    <!-- XTXT: QR Code error message: halfway body (10+ people already scanned)  -->
+    <string name="qr_code_error_max_person_threshold_body">"Die Corona-Warn-App darf nicht zur  Zertifikatsprüfung durch Dritte verwendet werden. Nutzen Sie dafür die CovPassCheck-App.\n\nSie können maximal für %s Personen Zertifikate hinzufügen. Danach können Sie keine neuen Zertifikate mehr für weitere Personen einscannen.\n\nWeitere Informationen zum  Impfzertifikat finden Sie in den FAQ."</string>
+    <!-- XTXT: QR Code error message: halfway title (10+ people already scanned)  -->
+    <string name="qr_code_error_max_person_max_title">"Fehler"</string>
+    <!-- XTXT: QR Code error message: halfway body (10+ people already scanned)  -->
+    <string name="qr_code_error_max_person_max_body">"Die Corona-Warn-App darf nicht zur Zertifikatsprüfung durch Dritte verwendet werden. Nutzen Sie dafür die CovPassCheck-App.\n\nSie können maximal für 20 Personen Zertifikate hinzufügen. Sie können keine neuen Zertifikate mehr für weitere Personen einscannen.\n\nWeitere Informationen zum  Impfzertifikat finden Sie in den FAQ."</string>
```
Comment (mtwalli) [inline Corona-Warn-App/src/main/res/values-de/strings.xml:957]: should be %s instead of 20

File: Corona-Warn-App/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -950,6 +950,22 @@
     <string name="qr_code_no_suitable_qr_code_title">"No Suitable QR Code"</string>
     <!-- XTXT: QR Code error message title  -->
     <string name="qr_code_no_suitable_qr_code_body">"The QR code is not supported by the Corona-Warn-App. Please select a suitable QR code."</string>
+    <!-- XTXT: QR Code error message: halfway title (10+ people already scanned)  -->
+    <string name="qr_code_error_max_person_threshold_title">"Warnung"</string>
+    <!-- XTXT: QR Code error message: halfway body (10+ people already scanned)  -->
+    <string name="qr_code_error_max_person_threshold_body">"Die Corona-Warn-App darf nicht zur  Zertifikatsprüfung durch Dritte verwendet werden. Nutzen Sie dafür die CovPassCheck-App.\n\nSie können maximal für 20 Personen Zertifikate hinzufügen. Danach können Sie keine neuen Zertifikate mehr für weitere Personen einscannen.\n\nWeitere Informationen zum  Impfzertifikat finden Sie in den FAQ."</string>
```
Comment (mtwalli) [inline Corona-Warn-App/src/main/res/values/strings.xml:956]: same here

**Review by KathrinLuecke — APPROVED** [review]
UA approved

**Review by chiljamgossow — COMMENTED** [review]
Tested with 20+ certificates and works as expected 🚀

**Review by mtwalli — APPROVED** [review]

**Review by chiljamgossow — APPROVED** [review]
🎉

**Pull request merged**